src/qt/walletframe.cpp

```diff
@@ -184,10 +190,38 @@ void WalletFrame::gotoVerifyMessageTab(QString addr)
 
 void WalletFrame::gotoLoadPSBT(bool from_clipboard)
 {
-    WalletView *walletView = currentWalletView();
-    if (walletView) {
-        walletView->gotoLoadPSBT(from_clipboard);
+    std::string data;
+    if (from_clipboard) {
+        std::string raw = QApplication::clipboard()->text().toStdString();
+        bool invalid;
+        data = DecodeBase64(raw, &invalid);
+        if (invalid) {
+            Q_EMIT message(tr("Error"), tr("Unable to decode PSBT from clipboard (invalid base64)"), CClientUIInterface::MSG_ERROR);
+            return;
+        }
+    } else {
+        QString filename = GUIUtil::getOpenFileName(this,
+                                                    tr("Load Transaction Data"), QString(),
+                                                    tr("Partially Signed Transaction (*.psbt)"), nullptr);
```

Comment:
I understand these indentations were suggested by `clang-format-diff.py`, but to keep verifying easy with `diff --color-moved=dimmed-zebra`, I'd suggest to keep the same indentations as they were in `walletview.cpp`.